main.go

```diff
@@ -107,6 +107,7 @@ func Init() {
 }
 
 func versionInfo() map[string]interface{} {
+	stats.NewTaggedStat("rudder_server_config", stats.GaugeType, stats.Tags{"version": version, "Major": major, "Minor": minor, "Patch": patch, "Commit": commit, "BuildDate": buildDate, "BuiltBy": builtBy, "GitUrl": gitURL, "TransformerVersion": transformer.GetVersion(), "DatabricksVersion": misc.GetDatabricksVersion()}).Gauge(1)
```

Comment:
we only need to push this metric once during startup, right?
```suggestion
	stats.NewTaggedStat("rudder_server_config", stats.GaugeType, stats.Tags{"version": version, "major": major, "minor": minor, "patch": patch, "commit": commit, "buildDate": buildDate, "builtBy": builtBy, "gitUrl": gitURL, "TransformerVersion": transformer.GetVersion(), "DatabricksVersion": misc.GetDatabricksVersion()}).Gauge(1)
```

Reply:
Here, I am assuming that to change any of these we need to restart the server. So, pushing it only once.

Reply:
The function `versionInfo` is called whenever someone hits /version endpoint. So, there is not guarantee that this will be pushed once.